produce/lib/produce/merchant.rb

```diff
@@ -0,0 +1,83 @@
+require 'spaceship'
+
+module Produce
+  class Merchant
+    def create(_options, _args)
+      login
+
+      merchant_identifier = Produce.config[:merchant_identifier] || UI.input("Merchant identifier (reverse-domain name style string starting with 'merchant'): ")
+
+      if merchant_exists? merchant_identifier
```

Comment:
Haha fixed. I was copying the style in group.rb [here](https://github.com/fastlane/fastlane/blob/master/produce/lib/produce/group.rb#L12) but agree the parentheses is much more readble